docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -951,6 +952,19 @@ The `--disable-multiple-windows` option disables support for multi-window testin
 
 *Related configuration file property*: [disableMultipleWindows](configuration-file.md#disablemultiplewindows).
 
+### --retry-test-pages
+
+If this option is enabled, TestCafe retries failed network requests for webpages visited during tests. The retry limit is ten times.
```

Comment:
```suggestion
If this option is enabled, TestCafe retries failed network requests for webpages visited during tests. The retry function is limited to ten tries. 
```

Reply:
`function` can mislead because there is an [entity called `function` in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Functions). Does this work:

If this option is enabled, TestCafe retries failed network requests for webpages visited during tests up to ten times.

Reply:
the last option is best